### ISSUE TITLE
Error Report: Attempt to invoke virtual method 'void org.apache.http.impl.conn.SingleClientConnManager$ConnAdapter.detach()' on a null object reference

### ISSUE THREAD

**Issue by rastating** [rastating/DroidBeard#282]
## Configuration Information
**Version:**  1.4.7
**HTTPS Enabled:**  false
**Trust All Certificates:**  false
## Stack Trace
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void org.apache.http.impl.conn.SingleClientConnManager$ConnAdapter.detach()' on a null object reference
	at org.apache.http.impl.conn.SingleClientConnManager.revokeConnection(SingleClientConnManager.java:396)
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:219)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.executeOriginal(DefaultRequestDirector.java:1258)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:700)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:691)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:514)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:492)
	at com.rastating.droidbeard.net.SickbeardAsyncTask.getJson(SickbeardAsyncTask.java:182)
	at com.rastating.droidbeard.net.FetchHistoryTask.doInBackground(FetchHistoryTask.java:40)
	at com.rastating.droidbeard.net.FetchHistoryTask.doInBackground(FetchHistoryTask.java:31)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)

```
## Data
```

```
